src/ssl.cxx

```diff
@@ -68,19 +83,40 @@ SSLMod::SSLMod(evhttp* http, const char* extip, bool enable)
 	unsigned char sha256_buf[32];
 	unsigned int i;
 
+# if OPENSSL_VERSION_NUMBER < 0x10100000L
 	SSL_load_error_strings();
 	SSL_library_init();
+# endif
 
 	std::unique_ptr<EVP_PKEY, std::function<void(EVP_PKEY*)>>
 		pkey{EVP_PKEY_new(), EVP_PKEY_free};
 	std::unique_ptr<X509, std::function<void(X509*)>>
 		x509{X509_new(), X509_free};
 	/* XXX: settable params */
 	std::unique_ptr<RSA, std::function<void(RSA*)>>
+# if OPENSSL_VERSION_NUMBER < 0x10000000L
 		rsa{RSA_generate_key(2048, RSA_F4, key_progress_cb, 0), RSA_free};
 
 	if (!pkey || !x509 || !rsa)
```

Comment:
Please don't mix part of one statement with followup statements in `#if` block, this makes it really hard to read. I.e. start the `#if` line earlier and include whole `std::unique_ptr` in it.